state/cockroachdb/cockroachdb_access.go

```diff
@@ -396,6 +406,13 @@ func (p *cockroachDBAccess) Query(req *state.QueryRequest) (*state.QueryResponse
 
 // Ping implements database ping.
 func (p *cockroachDBAccess) Ping() error {
+	for i := 0; i < maxRetries; i++ {
```

Comment:
This might retry too quickly. Please use `dapr/kit/retry` to make the retries also exponentially.
https://github.com/dapr/kit/blob/main/retry/retry.go